config/settings.yml

```diff
@@ -152,11 +152,6 @@
     :addition:
       :critical:
       - CloneTaskEvent
```

Comment:
yeah, thats why I only move events we _know_ that are coming from vmware, the remaining ones are cruft 😄